powershell/.gitignore

```diff
@@ -0,0 +1,2 @@
+extractor/**/bin/*
+extractor/**/obj/*
```

Comment:
😍 I've been meaning to do this for a while but never got around to it. Thank you!